.github/workflows/release.yml

```diff
@@ -12,6 +12,9 @@ on:
       homebrew-prerelease:
         description: 'HomeBrew prerelease'
         default: 'false'
+      draft-release:
```

Comment:
Input parameter flag to use the same workflow for a draft release and then an actual release.